GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -499,6 +499,26 @@ public override HashSet<string> UpgradeBlockingProcesses
 
             // Tests show that 250 is the max supported pipe name length
             public override int MaxPipePathLength => 250;
+
+            public override string UpgradeInstallAdviceMessage
+            {
+                get { return "When ready, run `" + this.UpgradeConfirmCommandMessage + "` from an elevated command prompt."; }
+            }
+
+            public override string UpgradeConfirmCommandMessage
+            {
+                get { return "`gvfs upgrade --confirm`"; }
+            }
+
+            public override string StartServiceCommandMessage
+            {
+                get { return $"`sc start GVFS.Service`";  }
+            }
+
+            public override string ElevatedCommandPromptMessage

```

Comment:
Do we still need `ElevatedCommandPromptMessage`?

If `UpgradeConfirmCommandMessage` were updated to return:

`"'gvfs upgrade --confirm'  from an elevated command prompt."`

Similar to how it returns the following on Mac:

`"'sudo gvfs upgrade --confirm'"`

Then we could probably remove both `ElevatedCommandPromptMessage` and `UpgradeInstallAdviceMessage` from the platform.

Reply:
If we do still need `ElevatedCommandPromptMessage` can you update the name?  I think this property used to return something like "from an elevated command prompt" and now it's returning a different message.

Reply:
`UpgradeConfirmCommandMessage` is also used in an error message for checking if the service is installed and not running, so we can't update that one.

The windows message does refer to an elevated prompt.  I'm updating it to be `RunUpdateMessage `which should be more clear.

Also as an FYI, potentially it's a bit of scope creep but @jamill and I went thru the messages and simplified what was there.  This should make life easier in the future.